clang/lib/CodeGen/CGObjCMac.cpp

```diff
@@ -3397,53 +3309,53 @@ void CGObjCMac::GenerateCategory(const ObjCCategoryImplDecl *OCD) {
 
 enum FragileClassFlags {
   /// Apparently: is not a meta-class.
-  FragileABI_Class_Factory                 = 0x00001,
+  FragileABI_Class_Factory = 0x00001,
```

Comment:
These are bit masks, and the horizontal alignment is helpful for seeing how they (don't) overlap; I would prefer to keep it.

Reply:
I can guard it with `clang-format off`.